gui/widgets/grid.cpp

```diff
@@ -213,23 +242,25 @@ GridItemTray::GridItemTray(GuiObject *boss, int x, int y, int w, int h, int entr
 	int trayPaddingX = buttonSpacingX;
 	int trayPaddingY = buttonSpacingY;
 
-	PicButtonWidget *playButton = new PicButtonWidget(this, trayPaddingX, trayPaddingY,
+	_playButton = new PicButtonWidget(this, trayPaddingX, trayPaddingY,
 													  2 * buttonWidth + buttonSpacingX, buttonHeight,
```

Comment:
Alignment seems to have gone wrong here.

Reply:
Fixed in 300b2ca.